include/misc/sflist.h

```diff
@@ -314,7 +314,7 @@ static inline void sys_sfnode_flags_set(sys_sfnode_t *node, u8_t flags)
  */
 static inline bool sys_sflist_is_empty(sys_sflist_t *list);
 
-Z_GENLIST_IS_EMPTY(sflist);
+Z_GENLIST_IS_EMPTY(sflist)
```

Comment:
Removing all those semis from the top level is just asking to blow up our poor editors' indentation logic.  But I guess these are code-generation macros, so...

Reply:
Macro hackery always deserves some punishment ;-)